qiskit/ignis/verification/randomized_benchmarking/dihedral.py

```diff
@@ -333,53 +338,53 @@ def __str__(self):
 
 class CNOTDihedral():
```

Comment:
In a future PR we should try and make this conform to the `BaseOperator` API like the `Clifford` class.

Reply:
I agree, however we still miss some important methods.